diff -ruN RavenSource_v3.6/Makefile RavenSource_v3.6-patch/Makefile
--- RavenSource_v3.6/Makefile	2023-01-26 17:14:12.327995835 -0500
+++ RavenSource_v3.6/Makefile	2023-01-26 17:20:00.256887397 -0500
@@ -12,19 +12,17 @@
 #            Jan 2019    Alan J. Barton - added OS name to created exe name
 ###
 # appname := raven_rev$(shell svnversion -n . | sed -e 's/:/-/g')_$(shell uname -o).exe
-appname := Raven.exe
+appname := raven
 
-CXX      := g++
-CXXFLAGS := -Wno-deprecated
-# some compilers require the c++11 flag, some may not
-CXXFLAGS += -c++11
+override CXXFLAGS := $(CXXFLAGS) -Wno-deprecated
 
 srcfiles := $(shell find . -name "*.cpp")
 objects  := $(patsubst %.cpp, %.o, $(srcfiles))
 
 # include netcdf
-CXXFLAGS += -Dnetcdf                # if netcdf is installed use "CXXFLAGS += -Dnetcdf",                    else "CXXFLAGS += "
-LDLIBS   := -L/usr/local -lnetcdf   # if netcdf is installed give first path "-L<PATH>"and then "-lnetcdf", else "LDLIBS   := "
+# if netcdf is installed use "CXXFLAGS += -Dnetcdf",                    else "CXXFLAGS += "
+override CXXFLAGS += -Dnetcdf $(CXXFLAGS)
+LDLIBS   := $(shell nc-config --libs)
 
 # if you use a UNIX system, uncomment the LDFLAGS line below
 # this is to allow for use a 1Gb, see http://linuxtoosx.blogspot.ca/2010/10/stack-overflow-increasing-stack-limit.html
@@ -33,7 +31,7 @@
 all: $(appname)
 
 $(appname): $(objects)
-	$(CXX) $(CXXFLAGS) -o $(appname) $(objects) $(LDLIBS) $(LDFLAGS) 
+	$(CXX) $(CXXFLAGS) -o $(appname) $(objects) $(LDLIBS) $(LDFLAGS)
 
 depend: .depend
