diff -ruN Raven-rev322/Makefile Raven-rev322-patch/Makefile
--- Raven-rev322/Makefile	2019-07-11 16:01:31.000000000 -0400
+++ Raven-rev322-patch/Makefile	2022-02-14 16:39:32.107946954 -0500
@@ -12,17 +12,16 @@
 #            Jan 2019    Alan J. Barton - added OS name to created exe name
 ###
 # appname := raven_rev$(shell svnversion -n . | sed -e 's/:/-/g')_$(shell uname -o).exe
-appname := Raven.exe
+appname := raven
 
-CXX      := g++
-CXXFLAGS := -Wno-deprecated
+override CXXFLAGS := $(CXXFLAGS) -Wno-deprecated
 
 srcfiles := $(shell find . -name "*.cpp")
 objects  := $(patsubst %.cpp, %.o, $(srcfiles))
 
-# include netcdf
-CXXFLAGS += -Dnetcdf                # if netcdf is installed use "CXXFLAGS += -Dnetcdf",                    else "CXXFLAGS += "
-LDLIBS   := -L/usr/local -lnetcdf   # if netcdf is installed give first path "-L<PATH>"and then "-lnetcdf", else "LDLIBS   := "
+# if netcdf is installed use "CXXFLAGS += -Dnetcdf",                    else "CXXFLAGS += "
+override CXXFLAGS := -Dnetcdf $(CXXFLAGS)
+LDLIBS   := $(shell nc-config --libs)
 
 # if you use a UNIX system, uncomment the LDFLAGS line below
 # this is to allow for use a 1Gb, see http://linuxtoosx.blogspot.ca/2010/10/stack-overflow-increasing-stack-limit.html
@@ -31,19 +30,10 @@
 all: $(appname)
 
 $(appname): $(objects)
-	$(CXX) $(CXXFLAGS) -o $(appname) $(objects) $(LDLIBS) $(LDFLAGS)
-
-depend: .depend
-
-.depend: $(srcfiles)
-	rm -f ./.depend
-	$(CXX) $(CXXFLAGS) -MM $^>>./.depend;
+	$(CXX) $(CXXFLAGS) -o $(appname) $(objects) $(LDLIBS) $(LDFLAGS)

 clean:
 	rm -f $(objects)
 
 dist-clean: clean
 	rm -f *~ .depend
-
-include .depend
-
